### PR TITLE
Compatibility with Latte 2.11 & 3

### PR DESCRIPTION
**Pull request by dg**

### CODE REVIEW

File: src/templates/datagrid.latte [hunk 1]
```diff
@@ -413,7 +413,7 @@
 
 {define inlineAddRow}
 	{if $inlineAdd->shouldBeRendered()}
-		{php $inlineAdd->onSetDefaults($filter['inline_add']); }
+		{php $inlineAdd->onSetDefaults($filter['inline_add']) }
```
Comment (petrparolek) [inline src/templates/datagrid.latte:416]: @dg in new Latte is semicolon is not allowed?

**Comment by s1t2e3** [comment]
merge and release...asap

**Comment by dg** [comment]
@s1t2e3 not yet, Latte 3 is still in beta.

**Comment by dg** [comment]
I think it's ready to be merged.

**Comment by paveljanda** [comment]
Thanks!

**Pull request merged**